draft-ietf-dnssd-privreq.xml

```diff
@@ -817,7 +824,128 @@ This draft does not require any IANA action.
+        derived based on time so that commitment proofs may be precomputed asynchronously. 
+        This avoids O(N*M) computation, where N is the number of nodes in a local network 
+        and M is the number of per-node pairings.</t>
+
+      <t>
+        This system has the following properties:
+        <list style='numbers'>
+          <t>Symmetric work load: clients and servers can pre-compute private instance names
+            as a function of their pairing secret and predictable nonce.</t>
+          <t>Mutual identity privacy: Both client and server identities are hidden from 
+            active and passive attackers that do not subvert the pairing process.</t>
+          <t>No client set size hiding: The number of private instance names
+          reveals the number of unique pairings a server has with its clients.</t>
+          <t>Unlinkability: Private service names are unlinkable to post-discovery TLS 
+            connections. (Note that if deterministic nonces repeat, servers risk linkability
+            across private service names.)</t>
+          <t>No fingerprinting: Assuming servers use fresh nonces per private instance name, 
+            advertisements change regularly.</t>
+        </list>
+      </t>
+    </section>
+
+    <section title="Private IoT">
+      <t>Boneh et al. <xref target="Wu16"></xref> developed an approach for private service 
+      discovery that reduces to private mutual authentication. Moreover, it should be 
+      infeasible for any adversary to forge advertisements or impersonate anyone else on 
+      the network. Specifically, service discoverers only wish to reveal their identity 
+      to services they trust, and vice versa. Existing protocols such as TLS, IKE, and 
+      <xref target="SIGMA">SIGMA</xref> require that one side reveal its identity first. 
+      Their approach first allocates, via some policy manager, key pairs associated with 
+      human-readable policy names. For example, user Alice might have a key pair associated 
+      with the names /Alice, /Alice/Family, and /Alice/Device. Her key is bound to each 
+      of these names. Authentication policies (and trust models) are then expressed as 
+      policy prefix patterns, e.g., /Alice/*. Broadcast messages are encrypted to policies. 
+      For example, Alice might encrypt a message m to the policy /Bob/*. Only Bob, who 
+      owns a private key bound to, e.g., /Bob/Devices, can decrypt m. (This procedure 
+      uses a form of identity-based encryption called prefix-based encryption. Readers 
+      are referred to <xref target="Wu16"></xref> for a thorough description.)</t>
+
+      <t>Using prefix- and policy-based encryption, service discovery is decomposed into 
+        two steps: (1) service announcement and (2) key exchange, similar to 
+        <xref target="I-D.huitema-dnssd-privacy"></xref>. Announcements carry service 
+        identities, ephemeral key shares, and a signature, all encrypted under the 
```

Comment:
The reference should be to I-D.ietf-dnssd-privacy.

Reply:
Will fix.

Reply:
Done.